### ISSUE TITLE
Compiling on Windows?

### ISSUE THREAD

**Issue by lonnietc** [mbitsnbites/resseract#2]
Hello,

Hope that you are doing well today, my friend.

I wanted to give you an update on things as recently, I was able to compile and run Resseract on Linux.

With that in mind, I wanted to ask you how complete was the port to C++ from the original Tesseract and also what are your thoughts on trying to port this to Windows 10 to have a Visual Studio 2022 project solution.

My immediate goals are to see it running on Windows and if the original port from Tesseract is pretty complete (i.e. functional networking, editing, speed, etc.) then I would like to then move to investigate replacing the current rendering engine with a modern cross-platform rendering framework as well as to explore the Octree for a new "infinite" solution that I have come across.

Basically, the idea is to take Resseract and try to build (slowly and methodically) a modern 3D engine to compete with state-of-the-art engines, over time of course and will also focus highly on things like a real-time ray-tracing integration for photorealistic environments in the future.

Maybe we can discuss it more together, but for now I would like to get it to compile and run on Windows as well.

Any thoughts?
Thanks and have a great day

**Comment by lonnietc** [comment]
Hello,

Just an update, but I almost have a Visual Studio 2022 solution working on Windows 10.

Getting one linker error that I cannot figure out yet.

```
1>server.obj : error LNK2019: unresolved external symbol SDL_main referenced in function WinMain
1>..\bin64\resseract_debug.exe : fatal error LNK1120: 1 unresolved externals
1>Done building project "resseract.vcxproj" -- FAILED.
```

**Comment by mbitsnbites** [comment]
Hi! I don't think that I ever tried compiling for Windows. The error you are seeing appears to be related to SDL. Maybe you need to add libsdl to the project?
